label_studio/data_export/api.py

```diff
@@ -370,6 +371,22 @@ class ExportDetailAPI(generics.RetrieveDestroyAPIView):
     lookup_url_kwarg = 'export_pk'
     permission_required = all_permissions.projects_change
 
+    def delete(self, *args, **kwargs):
+        if flag_set('ff_back_dev_4664_remove_storage_file_on_export_delete_29032023_short'):
+            try:
+                export = self.get_object()
+                export.file.delete()
```

Comment:
seems it doesn't delete converted files?

Reply:
I didn't know that we were saving converted files to storage. It doesn't seem like we save any reference to the converted files so they're hard to locate in this process.

I'm thinking we postpone doing the converted files deletion until after https://github.com/heartexlabs/label-studio/pull/3899/ is merged. Sergei's async changes store references to the converted files in the database now so it'll be much easier to do deletion that way.

Thoughts?

Reply:
Converted files deletion also implemented.